Algorithms/Java/towerOfHanoi.java

```diff
@@ -0,0 +1,32 @@
+class towerOfHanoi{
```

Comment:
It doesn't really matter, but kinda bothers, lets keep the class name in `PascalCase`